astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -1047,10 +1057,10 @@ def test_int_out_of_range(parallel):
     expected = Table([[12.3, 10.*imax], [10.*imax, 4.56e8]],
                      names=('A', 'B'))
 
-    table = ascii.read(text, format='basic', guess=False,
+    table = ascii.read(text, format='basic',
                        fast_reader={'parallel': parallel})
     assert_table_equal(table, expected)
-    table = ascii.read(text, format='basic', guess=False, fast_reader=False)
+    table = ascii.read(text, format='basic', fast_reader=False)
     assert_table_equal(table, expected)
 
 
```

Comment:
Yes, they are, the `guess=False` simply was not longer necessary. But in the spirit of rather adding than changing tests, it would probably be better to `@pytest.mark.parametrize('guess', [ False, True ])` this.
I may have been a bit too cavalier about these tests because I had only added the lot in 1.3 and expected this to go in soon afterwards...